Branch xiao kira ERP

kiraxjj opened the pull request.

kiraxjj committed:
ERP Page Created

Initial Edit on ERP

kiraxjj committed:
.yml Update

Update the .yml with ERP link

kiraxjj committed:
Update ERP.md

added Data Type

kiraxjj committed:
Add cycles and formating

Added revenue cycle, procurement cycle, and some formatting

kiraxjj committed:
Clickable video img, Website Title

Add Clickable video img to link to the video

Add Website Title instead of using plain urls

tag commented:
Please remove all of the manual `<br>` tags. Let Markdown handle the spacing; don't force it.

Please also do pull requests to a branch, rather than to tag/master. I'd like these edited and reviewed before I merge to master.

tag commented:
@kiraxjj I've changed the target branch. When you finish the edits to the ERP files, add those to the pull request, and I'll run it.

kiraxjj committed:
Update index. Update business Cycles, update ERP

- adjust formatting on ERP
- make Business Cycles an independent module
- adjust formatting on Bus Cycles
- Add the two modules into Index file
- Add more info into Bus Cycles

kiraxjj committed:
Finish Bus Cycle, ERP, except 1 error

Business Cycles and ERP modules are finished.

(One error in retrieving picture using relative path in Bus Cycle need to be fixed)

kiraxjj committed:
Update index.md

Fixed formatting error on index

kiraxjj committed:
Add IT Control draft 1

also tried to fix index, again

kiraxjj committed:
Update Business Cycles.md

Add in another practice

kiraxjj committed:
Update index.md

Finally, fixed index formatting issue

kiraxjj committed:
Finish IT Control (pic insert error)

Finish IT Contro Module except the inserted pic bug

kiraxjj committed:
Bus Cycle New Exercise

Added 1 more practice to Bus Cycle

kiraxjj committed:
ERP editted

kiraxjj committed:
Update ERP.md

kiraxjj committed:
Edit Business Cycles page, Added a Revenue Cycle Diagram

kiraxjj committed:
Edit the three pages

kiraxjj committed:
Update IT Control.md

kiraxjj committed:
Update mkdocs.yml

Put hyperlinks for modules in

kiraxjj committed:
Update ERP.md

Exercise question format
